crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -33,11 +33,11 @@ def initialize(backup)
     def upgrade
       case @version
       when "1.9"
-        knife_files_1_9
+        upgrade_knife_files_1_9
         reset_target_platform_for_nodes
-        crowbar_files_1_9
+        upgrade_crowbar_files_1_9
       when "3.0"
-        network_json_3_0
+        create_network_json_3_0
```

Comment:
The JSON file that is there is not modified by the user; it's really about creating it from what is in the backup, so I feel the name is correct. I could change to recreate_ if it really makes a difference...